### PR TITLE
Logo #1056

### LINKED ISSUE #1056: Design Logo
Design Logo for Website

### PR DESCRIPTION
**Pull request by ishitakapoor26**
Fixes #1056

### CODE REVIEW

**Comment by ishitakapoor26** [comment]
@AbhipsaGuru1012 pls review pr.

**Comment by smaranjitghose** [comment]
Along with the gif could you please provide an SVG for the same?

**Comment by ishitakapoor26** [comment]
> Along with the gif could you please provide an SVG for the same?

yeah sure but animations won't work in svg format!

**Comment by smaranjitghose** [comment]
> > Along with the gif could you please provide an SVG for the same?
> 
> yeah sure but animations won't work in svg format!

Ya, I know. This is basically for other places where we can't have a gif file. Also, update the favicon for the page with this

**Comment by ishitakapoor26** [comment]
I'm facing some issue in updating favicon to page, @AbhipsaGuru1012 can you please help?

**Comment by AbhipsaGuru1012** [comment]
> I'm facing some issue in updating favicon to page, @AbhipsaGuru1012 can you please help?

@ishitakapoor26 yes please let me know what issue you are facing

File: public/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     content="A student's savior to submit those assignments by just typing them out and getting it handwritten on the go!"
     data-react-helmet="true" />
   <link rel="preconnect" href="https://api.github.com">
-  <link rel="dynamic icon" id="favicon" href="doc2penlogo.png" />
+  <link rel="dynamic icon" id="favicon" href="src/assets/logo/DOC2PEN-favicon.gif" />
```
Comment (smaranjitghose) [inline public/index.html:13]: Update this in the seo component built using react-helmet

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Review by Ayushparikh-code — APPROVED** [review]

**Comment by anushbhatia** [comment]
logo is too small so the text present inside is barely visible. Also in small devices it might not be that prominent so can you do something for that if possible.

**Comment by ishitakapoor26** [comment]
> logo is too small so the text present inside is barely visible. Also in small devices it might not be that prominent so can you do something for that if possible.

okay will make some changes to it!

**Pull request closed without merging**